pkg/plugins/storage/fs.go

```diff
@@ -63,7 +73,9 @@ func (fs *FS) Extract(ctx context.Context, pluginID string, pluginArchive *zip.R
 }
 
 func (fs *FS) extractFiles(_ context.Context, pluginArchive *zip.ReadCloser, pluginID string) (string, error) {
-	installDir := filepath.Join(fs.pluginsDir, pluginID)
+	randSuffix := randomString(8)
```

Comment:
a bit of nit but do we need the random suffix here? (vs using just the pluginID temporarily)

Reply:
Yeah that's fair. I can change this in a follow up step where it makes more sense